tests/framework/installer/ceph_installer.go

```diff
@@ -929,6 +934,7 @@ func (h *CephInstaller) GatherAllRookLogs(testName string, namespaces ...string)
 	for _, namespace := range namespaces {
 		h.k8shelper.GetLogsFromNamespace(namespace, testName, utils.TestEnvName())
 		h.k8shelper.GetPodDescribeFromNamespace(namespace, testName, utils.TestEnvName())
+		h.k8shelper.GetPodRestartsFromNamespace(namespace, testName, utils.TestEnvName())
```

Comment:
Won't this result in duplicate error checking from the check already on line 638?

Reply:
Yaa removing it from here